assets/src/application/ui/input/EspressoButton/types.ts

```diff
@@ -21,16 +21,16 @@ export enum EspressoButtonType {
 	SECONDARY = 'SECONDARY',
 }
 
-export interface EspressoButtonProps extends Partial<NativeButtonProps> {
-	buttonProps?: object;
+export interface EspressoButtonProps
+	extends Partial<NativeButtonProps>,
+		Partial<withLabelProps>,
+		Partial<withTooltipProps> {
```

Comment:
because label and tooltip props can be passed to the button but will get intercepted by the HOCs